src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./Navbar.css";
+
+function Navbar() {
+  return (
+    <nav className="nav-bar navbar-container mb-0">
+      <div className="nav-innerContainer font-clr">
+        <h2 className="nav-heading mr-4 nav-custom-sty">Social Cloud.</h2>
+      </div>
+
+      <div className="nav-innerContainer font-clr width-auto">
+        <input className="nav_searchBar" type="text" />
+        <span className="searchBar_icon">
+          <i className="fa-solid fa-magnifying-glass"></i>
```

Comment:
Try using react-icons package, it contains a lot more icons and all free.

Reply:
Sure I will look into it